### ISSUE TITLE
Considerable underperformance on AIM (non-salient and Transparent object types).

### ISSUE THREAD

**Issue by xin1u240** [naver-ai/ZIM#23]
I evaluated ZIM on some common matting dataset AM2K, P3M, and AIM, and I found quite different metrics compared with reported in the paper especially for the latter two (around 57 on P3M). I also visualize some bad cases and find this method fails on semantically ambiguous or underrepresented subjects such like flames, water, smoke, starry skies, spiderwebs, shrubs, wire fences, mesh barriers, and even simple portrait. Below are some examples. It's hard to imagine that a SAM-based zero shot method could perform so poorly on such widely used public datasets. 

| Pred| GT | Input |
| :---: | :---: | :---: |
| <img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/514aef08-10dd-4ca6-b61f-155caa6a6bb0" />|<img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/c2db7fa6-a464-400b-80ad-04928eba1236" />|<img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/ee403ab0-30ae-4493-9bbd-30d54657dd2c" /> |
| <img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/613bc7c9-4ab2-47a6-bef2-ff097ff68aa4" />|<img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/6f5a24db-8982-4999-83c0-3b93b9362efd" />|<img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/fb6482f6-00d4-4b24-916d-ce9ac887d64d" /> |
| <img width="150" height="260" alt="Image" src="https://github.com/user-attachments/assets/df6405ec-6b84-4e35-a476-82ed1f858be8" />|<img width="150" height="260" alt="Image" src="https://github.com/user-attachments/assets/1be21208-b2ae-4af9-a208-763bdcb89708" />|<img width="150" height="260" alt="Image" src="https://github.com/user-attachments/assets/a4d9de1f-c37f-4313-b48b-4e060c628ace" />|
| <img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/bba73ce2-862b-48bf-987f-8dc45252bc4c" />|<img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/254c570a-dfbe-4f36-916e-f1f0ce84241e" />|<img width="260" height="150" alt="Image" src="https://github.com/user-attachments/assets/8d517c8f-5243-4494-92c7-5464df58ece6" />|

In my point of view the "matting" should work on general "foreground", otherwise the method should be categorized as "segmentation" method. Also, I find no publication of the training dataset and label converter. I hope my discoveries can be helpful for those who plan to follow this repo for work or reasearch.
